packages/qr-code-generator/src/types/IQRCodeGenerator.ts

```diff
@@ -49,6 +51,15 @@ export interface SIOPv2DataWithScheme {
   requestUri: string
 }
 
+export type OpenID4VCIScheme = 'openid-credential-offer' | 'https' | string
+export interface OpenID4VCIDataWithScheme {
+  scheme?: OpenID4VCIScheme
```

Comment:
Please replace domain and path with an optional baseUri

Reply:
Fixed!